### ISSUE TITLE
Bug: This version of Query Engine is not compatible with Prisma Client

### ISSUE THREAD

**Issue by thaddeuskkr** [dicedtomatoreal/typex#492]
### What happened?

Running the Docker image (latest version, v3.7.6) with PostgreSQL 15 produces this issue when attempting to log-in.
```
PrismaClientInitializationError: 
Invalid `prisma.user.findMany()` invocation:
This version of Query Engine is not compatible with Prisma Client: "ip" and "port" fields are missing in the startup log entry
    at xc.handleRequestError (/zipline/node_modules/@prisma/client/runtime/binary.js:232:7603)
    at xc.handleAndLogRequestError (/zipline/node_modules/@prisma/client/runtime/binary.js:232:6755)
    at xc.request (/zipline/node_modules/@prisma/client/runtime/binary.js:232:6345)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async c (/zipline/.next/server/pages/api/auth/login.js:1:2814) {
  clientVersion: '4.16.2',
  errorCode: undefined
}
```
This same issue does not occur on v3.7.5.

### Version

latest (ghcr.io/diced/zipline or ghcr.io/diced/zipline:latest)

### What browser(s) are you seeing the problem on?

Chromium-based (Chrome, Edge, Brave, Opera, mobile chrome/chromium based, etc)

### Zipline Logs

```shell
2023-11-21 08:00:43,306 AM info  [server::response] GET /api/user -> 401
PrismaClientInitializationError: 
Invalid `prisma.user.findMany()` invocation:


This version of Query Engine is not compatible with Prisma Client: "ip" and "port" fields are missing in the startup log entry
    at xc.handleRequestError (/zipline/node_modules/@prisma/client/runtime/binary.js:232:7603)
    at xc.handleAndLogRequestError (/zipline/node_modules/@prisma/client/runtime/binary.js:232:6755)
    at xc.request (/zipline/node_modules/@prisma/client/runtime/binary.js:232:6345)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async c (/zipline/.next/server/pages/api/auth/login.js:1:2814) {
  clientVersion: '4.16.2',
  errorCode: undefined
}
2023-11-21 08:00:45,219 AM info  [server::response] POST /api/auth/login -> 500
```

### Additional Info

I did not make any relevant configuration changes.

**Comment by Nollknolle** [comment]
Same problem here. Appearing since today

**Comment by wdhdev** [comment]
I'm getting this as well; I cannot sign into the dashboard.
The newest version seems to have broken.

**Comment by TheZoker** [comment]
Same here

I downgraded to the previous version and everything works again

**Comment by ryzetech** [comment]
Same here, was wondering if I did something wrong while installing

**Comment by Chloe-ko** [comment]
Same here, started with the new update

**Comment by diced** [comment]
Sorry for the bug, for anyone having this issue please provide some extra info for me :)

- Output of `docker compose exec zipline yarn exec prisma version` or `yarn prisma version`

**Comment by TheZoker** [comment]
This is the output of `docker compose exec zipline yarn exec prisma version` (using the latest version of the zipline docker image):
```
prisma                : 4.16.2
@prisma/client        : 4.16.2
Current platform      : linux-musl
Query Engine (Binary) : query-engine aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/query-engine, resolved by PRISMA_QUERY_ENGINE_BINARY)
Migration Engine      : migration-engine-cli aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/migration-engine, resolved by PRISMA_MIGRATION_ENGINE_BINARY)
Format Wasm           : @prisma/prisma-fmt-wasm 4.16.1-1.4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Default Engines Hash  : 4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Studio                : 0.484.0
```

**Comment by JustJoostNL** [comment]
Same issue here unfortunately, output of command below. Hope it helps :)

```
prisma                : 4.16.2
@prisma/client        : 4.16.2
Current platform      : linux-musl-arm64-openssl-1.1.x
Query Engine (Binary) : query-engine aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/query-engine, resolved by PRISMA_QUERY_ENGINE_BINARY)
Migration Engine      : migration-engine-cli aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/migration-engine, resolved by PRISMA_MIGRATION_ENGINE_BINARY)
Format Wasm           : @prisma/prisma-fmt-wasm 4.16.1-1.4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Default Engines Hash  : 4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Studio                : 0.484.0
```

**Comment by ryzetech** [comment]
Might be redundant, but I'll post my info as well :)

```
prisma                : 4.16.2
@prisma/client        : 4.16.2
Current platform      : linux-musl
Query Engine (Binary) : query-engine aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/query-engine, resolved by PRISMA_QUERY_ENGINE_BINARY)
Migration Engine      : migration-engine-cli aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/migration-engine, resolved by PRISMA_MIGRATION_ENGINE_BINARY)
Format Wasm           : @prisma/prisma-fmt-wasm 4.16.1-1.4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Default Engines Hash  : 4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Studio                : 0.484.0
```

**Comment by alexdelprete** [comment]
Ditto. :)

```console
prisma                : 4.16.2
@prisma/client        : 4.16.2
Current platform      : linux-musl
Query Engine (Binary) : query-engine aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/query-engine, resolved by PRISMA_QUERY_ENGINE_BINARY)
Migration Engine      : migration-engine-cli aead147aa326ccb985dcfed5b065b4fdabd44b19 (at ../prisma-engines/migration-engine, resolved by PRISMA_MIGRATION_ENGINE_BINARY)
Format Wasm           : @prisma/prisma-fmt-wasm 4.16.1-1.4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Default Engines Hash  : 4bc8b6e1b66cb932731fb1bdbbc550d1e010de81
Studio                : 0.484.0
```

**Comment by diced** [comment]
Not sure why this is happening since the versions do match the one in package.json :(

**Comment by alexdelprete** [comment]
> Not sure why this is happening since the versions do match the one in package.json :(

were you able to reproduce the issue?

**Comment by diced** [comment]
Nevermind, the binaries provided in the docker file are built on 4.10.x, so I guess 4.16 introduced something that broke it all

**Comment by diced** [comment]
Fixed in b4ec1088d1ec9472a6633d17b8d33c7cdd842300! There will be a 3.7.7 release shortly by tonight.
